tests/unit/utils/SignStreamProcess-test.js

```diff
@@ -13,12 +13,15 @@ describe('SignStreamProcessor', () => {
,"readability":"COMPLETE","line_name":"1","col_id":9967,"sign":"י","is_retraced":0,"is_reconstructed":1,"sign_id":1349754},{"is_retraced":0,"sign_id":1349755,"is_reconstructed":1,"next_sign_id":1349756,"readability":"COMPLETE","line_name":"1","line_id":45821,"sign":"ל","col_id":9967,"prev_sign_id":1349754,"col_name":"col. 1a","is_variant":0,"break_type":null},{"is_retraced":null,"sign_id":1349756,"is_reconstructed":null,"readability":null,"next_sign_id":1349757,"line_name":"1","line_id":45821,"sign":"ה","col_id":9967,"prev_sign_id":1349755,"col_name":"col. 1a","break_type":null,"is_variant":0},{"next_sign_id":1349758,"readability":null,"line_name":"1","line_id":45821,"sign":"","col_id":9967,"is_retraced":null,"sign_id":1349757,"is_reconstructed":null,"prev_sign_id":1349756,"col_name":"col. 1a","break_type":null,"is_variant":0},{"sign":"ו","col_id":9967,"next_sign_id":1349759,"readability":null,"line_name":"1","line_id":45821,"is_reconstructed":null,"sign_id":1349758,"is_retraced":null,"col_name":"col. 1a","is_variant":0,"break_type":null,"prev_sign_id":1349757},{"prev_sign_id":1349758,"col_name":"col. 1a","break_type":null,"is_variant":0,"is_retraced":null,"is_reconstructed":null,"sign_id":1349759,"next_sign_id":1349760,"readability":null,"line_name":"1","line_id":45821,"sign":"א","col_id":9967},{"is_retraced":0,"is_reconstructed":1,"sign_id":1349760,"line_id":45821,"line_name":"1","readability":"COMPLETE","next_sign_id":1349761,"col_id":9967,"sign":"ח","prev_sign_id":1349759,"break_type":null,"is_variant":0,"col_name":"col. 1a"},{"sign":"ר","col_id":9967,"readability":"COMPLETE","line_name":"1","next_sign_id":1349762,"line_id":45821,"sign_id":1349761,"is_reconstructed":1,"is_retraced":0,"col_name":"col. 1a","break_type":null,"is_variant":0,"prev_sign_id":1349760},{"col_name":"col. 1a","break_type":null,"is_variant":0,"prev_sign_id":1349761,"sign":"י","col_id":9967,"readability":"COMPLETE","next_sign_id":1349763,"line_name":"1","line_id":45821,"is_reconstructed":1,"sign_id":1349762,"is_retraced":0},{"col_name":"col. 1a","is_variant":0,"break_type":null,"prev_sign_id":1349762,"sign_id":1349763,"is_reconstructed":1,"is_retraced":0,"sign":"","col_id":9967,"line_name":"1","readability":"COMPLETE","next_sign_id":1349764,"line_id":45821},{"break_type":null,"is_variant":0,"col_name":"col. 1a","prev_sign_id":1349763,"sign_id":1349764,"is_reconstructed":1,"is_retraced":0,"col_id":9967,"sign":"","line_id":45821,"next_sign_id":1349765,"readability":"COMPLETE","line_name":"1"},{"col_name":"col. 1a","is_variant":0,"break_type":null,"prev_sign_id":1349764,"sign_id":1349765,"is_reconstructed":1,"is_retraced":0,"sign":"","col_id":9967,"next_sign_id":1349766,"readability":"COMPLETE","line_name":"1","line_id":45821},{"is_retraced":null,"is_reconstructed":null,"sign_id":1349766,"next_sign_id":null,"readability":null,"line_name":"1","line_id":45821,"sign":"","col_id":9967,"prev_sign_id":1349765,"col_name":"col. 1a","is_variant":0,"break_type":"LINE_END,COLUMN_END"}]
```

Comment:
Suggest putting this test data in a separate file (called `sign-stream-test-data.js` or something) and importing it into the test.

Reply:
Thanks for the hint.  I will make the update in the next pass, that would really clean things up.  can I just the use: const unprocessedSignStream = require('/path/to/json.file')?

Reply:
You should be able to use JSON... if not, we'll need to extend our Webpack config. That shouldn't be troublesome to do so just let me know if you get issues.